### PR TITLE
add description from api.gouv2.fr

### PR DESCRIPTION
**Pull request by ThibautGery**
Ajout d'une description dans api.gouv.fr
TODO : 
 * trouver une vraie image @MattiSG @Flightan 
 * retravailler le texte (voir #21) @ppezziardi @patama

description-apigouv

### CODE REVIEW

File: css/homepage.scss [hunk 1]
```diff
@@ -31,6 +31,67 @@
  }
 }
 
+#description {
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  .icon {
+    line-height: 1;
+    vertical-align: middle;
+    font-size: 20em;
```
Comment (MattiSG) [inline css/homepage.scss]: Specify `font-size` in percentage, not in `em`.

> `em` is relative to the font-size itself, it is less confusing to refer to it as a percentage.

File: css/homepage.scss [hunk 2]
```diff
@@ -31,6 +31,67 @@
...
+  ul {
+    display: flex;
+    font-size: 1.1em;
+    padding-top: 1em;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    li {
+      position: relative;
+      display: list-item;
```
Comment (MattiSG) [inline css/homepage.scss]: Why do you need to specify this on an `li`?

File: css/homepage.scss [hunk 3]
```diff
@@ -31,6 +31,67 @@
...
+          top: calc(50% - 2px);
+          width: 6px;
+          height: 6px;
+          background: #dde5eb;
+          box-shadow: inset 0 1px 1px rgba(127,153,176,.3);
+          border-radius: 3px;
+        }
+      }
+      a {
+        &:hover:before {
```
Comment (MattiSG) [inline css/homepage.scss]: > Please use the double-colon syntax for pseudo-elements  (`::before`, `::after`), as opposed to pseudo-selectors (`:hover`, `:first-child`…).

File: css/homepage.scss [hunk 4]
```diff
@@ -31,6 +31,67 @@
...
+          transform:none;
+          opacity:1;
+          transition-duration:.4s
```
Comment (MattiSG) [inline css/homepage.scss]: Spacing inconsistency.

File: css/homepage.scss [hunk 5]
```diff
@@ -31,6 +31,67 @@
...
+        }
+        &:before {
+          content: "";
+          position: absolute;
+          width: 100%;
+          height: 2px;
+          bottom: -4px;
+          background: #e5ebf0;
+          transform: translateY(7px);
```
Comment (MattiSG) [inline css/homepage.scss]: This all looks very fragile. Have you checked rendering when resizing the font of your browser?

File: css/homepage.scss [hunk 6]
```diff
@@ -31,6 +31,67 @@
...
+    li {
+      position: relative;
+      display: list-item;
+      &:first-child {
+        margin-right: 1.8em;
+        &:after {
+          content: "";
+          position: absolute;
+          right: -17px;
+          top: calc(50% - 2px);
```
Comment (MattiSG) [inline css/homepage.scss]: Degradability?

File: index.html [hunk 7]
```diff
@@ -23,6 +23,30 @@ <h1 class="ui inverted header">
   </div>
 </section>
 
+<section id="description" class="ui container">
+  <div class="ui fluid centered stackable grid">
+    <div class="six wide column">
+      <p>
+        <i class="cloud download icon"></i>
```
Comment (MattiSG) [inline index.html]: IMO download ≠ API

File: index.html [hunk 8]
```diff
@@ -23,6 +23,30 @@ <h1 class="ui inverted header">
...
+      </p>
+    </div>
+    <div class="six wide column">
+      <h2>Pourquoi api.gouv ?</h2>
+      <p>Pour authentifier simplement un usager, pour le guider dans la saisie d'informations, pour ne pas lui demander des informations que l’administration possède déjà, pour personnaliser un service selon la situation de l’usager, ... autant de cas d'usage disponible via les API de l'Etat. A vous d'inventer la suite !</p>
```
Comment (MattiSG) [inline index.html]: - https://github.com/sgmap/bonnes-pratiques#points-de-suspension
- disponibleS
- `À vous`

File: index.html [hunk 9]
```diff
@@ -23,6 +23,30 @@ <h1 class="ui inverted header">
...
 
+<section id="description" class="ui container">
+  <div class="ui fluid centered stackable grid">
+    <div class="six wide column">
+      <p>
+        <i class="cloud download icon"></i>
+      </p>
+    </div>
+    <div class="six wide column">
+      <h2>Pourquoi api.gouv ?</h2>
```
Comment (MattiSG) [inline index.html]: `.fr` ?

File: css/homepage.scss (hunk 1 above)
Comment (ThibautGery) [inline css/homepage.scss]: I specified the size with the unit used by semantic-ui. Don't you think it's better to use the same metric ?

File: css/homepage.scss (hunk 5 above)
Comment (ThibautGery) [inline css/homepage.scss]: Yep It's fine

File: css/homepage.scss (hunk 6 above)
Comment (ThibautGery) [inline css/homepage.scss]: What do you mean ?

File: index.html (hunk 7 above)
Comment (ThibautGery) [inline index.html]: that's true, we should change it. I'm looking for the right one

File: index.html (hunk 8 above)
Comment (ThibautGery) [inline index.html]: Fixing the copy paste

File: index.html (hunk 9 above)
Comment (ThibautGery) [inline index.html]: We all use `api.gouv` `beta.gouv` without the `.fr`

Comment (MattiSG) [inline index.html]: Ok, just asking for an explicit jurisprudence  :wink:

File: css/homepage.scss (hunk 6 above)
Comment (MattiSG) [inline css/homepage.scss]: How will that behave for the [5% of users](http://caniuse.com/#feat=calc <http://caniuse.com/#feat=calc>) (most probably much more effective visitors considering the audience here) who don’t have `calc` support?

File: css/homepage.scss (hunk 1 above)
Comment (MattiSG) [inline css/homepage.scss]: Ok, if extending an existing ruleset consistency > style  :)

**Pull request merged**